docs/installation.md

```diff
@@ -26,9 +26,13 @@ To activate Unmock and start intercepting internet traffic, call `unmock.on()`.
 // import unmock from "unmock-node";  // ES6
 const unmock = require("unmock-node").default; // CommonJS
 
-beforeAll(() => unmock.on()); // Activate unmock to intercept all outgoing traffic
+beforeAll(() => {
+  unmock.on();
+}); // Activate unmock to intercept all outgoing traffic
```

Comment:
Can also use the one-liner notation without the curly braces.
`beforeAll() => unmock.on());` and `afterAll(() => unmock.off());` both work fine now.